keras/losses.py

```diff
@@ -1885,6 +1885,20 @@ def _ragged_tensor_sparse_categorical_crossentropy(y_true,
       sparse_categorical_crossentropy, from_logits=from_logits, axis=axis)
   return _ragged_tensor_apply_loss(fn, y_true, y_pred, y_pred_extra_dim=True)
 
+def _check_ground_truth_range(y_true):
+  """ Ensures ground truth labels in binary_crossentropy and 
+      binary_focal_crossentropy are either 0.0 or 1.0.
+  """
+  y_true= tf.reshape(y_true, [-1])
+  unique, idx, count= tf.unique_with_counts(y_true)
+  if unique.shape == 2: 
+    unique= tf.sort(unique)
+    if unique[0]== 0 and unique[1]== 1:
```

Comment:
I am not sure if this will do the correct thing when comparing the graph tensor, especially when the unique value are tf.floats